hs/MakeElements.hs

```diff
@@ -53,6 +53,13 @@ aboutLinks = concatHtml (map stylesheet ["//netdna.bootstrapcdn.com/bootstrap/3.
                                        "static/style/graph/styles.css",
                                        "static/style/common/about.css",
                                        "static/style/common/common.css"])
+drawLinks :: H.Html
+drawLinks = concatHtml (map stylesheet ["//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css",
+                                       "static/style/graph/styles.css",
+                                       "static/style/graph/graph_styles.css",
+                                       "static/style/common/about.css",
+                                       "static/style/common/common.css"])
```

Comment:
@christinem Merging `happstack` into ones project branch before/while making a pull request makes the pull request diff code very hard to understand. I don't think that this is a good idea.

When merging an issue into the happstack branch, though, merging before is advised.